### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by DariaSlayer**
- [DEMO LINK](https://DariaSlayer.github.io/layout_catalog/)
- [TEST REPORT LINK](https://DariaSlayer.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,320 @@
...
+            <a class="nav__link" href="#Smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item" >
+            <a
+              class="nav__link"
+              href="#laptopsAndComputers"
+              data-qa="nav-hover">
+              Laptops & Computers
+            </a>
```
Comment (xiimiik) [inline src/index.html:36]:
```suggestion
            <a
              class="nav__link"
              href="#laptopsAndComputers"
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
```

File: src/styles/main-tag.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```
Comment (xiimiik) [inline src/styles/main-tag.scss:30]: checklist point 10 - `Make use of SASS nesting`

File: src/styles/navigation.scss [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+.nav__item {
+  margin-left: 20px;
+
+  &:first-child {
+    margin-left: 0;
+  }
+}
+
+.nav__link {
```
Comment (xiimiik) [inline src/styles/navigation.scss:16]: checklist point 10 - `Make use of SASS nesting`

File: src/styles/stars.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  }
+}
+
+.stars--0 .stars__star:nth-child(n+1),
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(../images/star.svg);
+}
```
Comment (xiimiik) [inline src/styles/stars.scss:20]: checklist point 12 - `Try using for loops for the stars selector.`

File: src/styles/stars.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star-active.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
```
Comment (xiimiik) [inline src/styles/stars.scss:10]: last start shouldnt have margin

File: src/styles/resets.scss [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+  margin: 0;
+}
+
+h1 {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
```
Comment (xiimiik) [inline src/styles/resets.scss:11]: u can group it

File: src/styles/navigation.scss [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  color: #060835;
+  transition: color 0.3s;
+
+  &:hover {
+    color: $buttonColour;
+  }
+
+  &.is-active {
+    color: $buttonColour;
+  }
```
Comment (xiimiik) [inline src/styles/navigation.scss:34]: do not increase specifity, add color only for `is-active`

**Review by xiimiik — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 8]
```diff
@@ -4,9 +4,321 @@
...
+      </nav>
+    </header>
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          class="card__picture"
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27inches Retina 5k Monoblock"
+        >
+        <h1 class="card__product-title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```
Comment (mykhalenych) [inline src/index.html:65]: h1 can be only one at page

File: src/styles/card.scss [hunk 9]
```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  box-sizing: border-box;
```
Comment (mykhalenych) [inline src/styles/card.scss:7]: move to `*` selector and delete in all other places

File: src/styles/card.scss [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+    transform: scale(120%);
+  }
+
+  &__picture {
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-title {
```
Comment (mykhalenych) [inline src/styles/card.scss:22]:
```suggestion
  &__product {
    &-title {
      &:hover {}
    }
    &-code {}
```

File: src/styles/card.scss [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+    transition-duration: 300ms;
+
+    &:hover {
+      color: $buttonColour;
+      background-color: #fff;
+    }
+  }
+}
+
+.card:hover .card__product-title {
```
Comment (mykhalenych) [inline src/styles/card.scss:93]:
```suggestion
```

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Nice work, but fix small suggestions

File: src/styles/card.scss (hunk 10 above)
Comment (DariaSlayer) [reply src/styles/card.scss:22]: then the change of color will apply only to the title itself while hovering it. but the task says it should be applied while hovering card

**Review by xiimiik — APPROVED** [review]
Nice work!

**Pull request closed without merging**